src/main/steps/applicant1/hub-page/joint/content.ts

```diff
@@ -329,6 +404,35 @@ export const generateContent: TranslationFn = content => {
     .add(config.get('dates.changingToSolePartnerResponseDays'), 'day')
     .isBefore(dayjs());
 
+  const latestRequestForInformation = userCase.requestsForInformation?.at(0)?.value;
+  const jointParties = latestRequestForInformation?.requestForInformationJointParties;
+
+  const isJointRequestForInformationForBothParties = jointParties === 'both';
+  const isJointRequestForInformationForApplicant2 = jointParties === 'applicant2';
+  const isJointRequestForInformationForApplicant1 = jointParties === 'applicant1';
+
+  const isApplicant1AbleToRespondToRequestForInformation =
+    !isApplicant2 && (isJointRequestForInformationForApplicant1 || isJointRequestForInformationForBothParties);
+
+  const isApplicant1RequestForApplicant2 = !isApplicant2 && isJointRequestForInformationForApplicant2;
+
+  const isApplicant2AbleToRespondToRequestForInformation =
+    isApplicant2 && (isJointRequestForInformationForApplicant2 || isJointRequestForInformationForBothParties);
+
+  const isApplicant2RequestForApplicant1 = isApplicant2 && isJointRequestForInformationForApplicant1;
+
+  const isApplicantAbleToRespondToRequestForInformation =
+    isApplicant1AbleToRespondToRequestForInformation || isApplicant2AbleToRespondToRequestForInformation;
+
+  const isRequestForInformationForYourPartner = isApplicant1RequestForApplicant2 || isApplicant2RequestForApplicant1;
```

Comment:
```suggestion
  const requestForInformationParties = latestRequestForInformation?.requestForInformationJointParties;
  
  const isApplicantAbleToRespondToRequestForInformation =
    ['both', isApplicant2 ? 'applicant2' : 'applicant1'].includes(requestForInformationParties);

  const isRequestForInformationForYourPartner = requestForInformationParties === (isApplicant2 ? 'applicant1' : 'applicant2');
```

Reply:
Nice - minor change required to handle undefined string on line 408, but otherwise implemented.